app/views/mappings/index.html.haml

```diff
@@ -1,14 +1,23 @@
-- @title= t('mappings.title')
-%div.container
-  %div#mappings_container.container-fluid.py-4.flex-grow-1
-    %h1.my-1= t('mappings.title')
-
-    %div#mappings_uploader.my-2
-      = link_to_modal t('mappings.upload_mappings') , "/mappings/loader", class: "btn btn-primary btn-block",
-        data: { show_modal_title_value: t('mappings.mappings_bulk_load'), show_modal_size_value: 'modal-xl'}
-    %hr.my-3.w-100
-      #accordionExample.accordion
-        = render partial: 'ontology_mappings'
-        = render partial: 'concept_mappings_selector'
-
+.mappigs-page-container{'data-controller': 'mappings', 
+                        'data-mappings-mappings-list-value': "#{@ontologies_mapping_count.to_h.to_json}",
+                        'data-mappings-zoom-ratio-value': 1,
+                        'data-mappings-type-value': 'page'
```

Comment:
'data-mappings-type-value': 'page' and 'data-mappings-zoom-ratio-value': 1 can be set a default and removed here